docs/controls/TabbedCommandBar.md

```diff
@@ -0,0 +1,209 @@
+                </AppBarButton.Resources>
+            </AppBarButton>
+            <AppBarButton Icon="Setting">
+                <AppBarButton.Resources>
+                    <Thickness x:Key="AppBarButtonTextLabelOnRightMargin">0,11,12,13</Thickness>
+                </AppBarButton.Resources>
+            </AppBarButton>
+        </CommandBar>
+    </ribbon:TabbedCommandBar.Footer>
+
+    <ribbon:TabbedCommandBar.Resources>
+        <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="SystemControlChromeMediumLowAcrylicWindowMediumBrush" />
+        <StaticResource x:Key="TabbedCommandBarContentBackground" ResourceKey="SystemControlChromeLowAcrylicWindowBrush" />
+    </ribbon:TabbedCommandBar.Resources>
+
+    <ribbon:TabbedCommandBarItem Header="Home">
+        <AppBarButton Icon="Undo" Label="Undo"/>
+        <AppBarButton Icon="Redo" Label="Redo"/>
+        <AppBarButton Icon="Paste" Label="Paste"/>
+        <AppBarSeparator />
+        <AppBarElementContainer>
+            <ComboBox SelectedIndex="0">
+                <ComboBoxItem Content="Arial" />
+                <ComboBoxItem Content="Calibri" />
+                <ComboBoxItem Content="JetBrains Mono" />
+                <ComboBoxItem Content="Roboto" />
+                <ComboBoxItem Content="Sergio UI" />
+                <ComboBoxItem Content="Sergio UI Semibold" />
+            </ComboBox>
+        </AppBarElementContainer>
+        <AppBarToggleButton Icon="Bold" Label="Bold" />
+        <AppBarToggleButton Icon="Italic" Label="Italic" />
+        <AppBarToggleButton Icon="Underline" Label="Underline" />
+    </ribbon:TabbedCommandBarItem>
+
+    <ribbon:TabbedCommandBarItem Header="Insert">
+        <AppBarButton Icon="Pictures" Label="Pictures">
+            <AppBarButton.Flyout>
+                <MenuFlyout Placement="BottomEdgeAlignedLeft">
+                    <MenuFlyoutItem Text="This Device">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xEC4E;" />
+                        </MenuFlyoutItem.Icon>
+                    </MenuFlyoutItem>
+                    <MenuFlyoutItem Text="Stock Images">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE721;" />
+                        </MenuFlyoutItem.Icon>
+                    </MenuFlyoutItem>
+                    <MenuFlyoutItem Icon="Globe" Text="Online Pictures" />
+                </MenuFlyout>
+            </AppBarButton.Flyout>
+        </AppBarButton>
+
+        <ribbon:TabbedCommandBarItem.SecondaryCommands>
+            <AppBarButton Icon="Add" Label="New item" />
+        </ribbon:TabbedCommandBarItem.SecondaryCommands>
+    </ribbon:TabbedCommandBarItem>
+</ribbon:TabbedCommandBar>
+```
+
+## Sample Output
+
+ ![TabbedCommandBar Overview](../resources/images/Controls/TabbedCommandBar/Overview.gif)
```

Comment:
I am unable to find a `TabbedCommandBar/Overview.gif` path in the `docs/resources/images/Controls` folder.